scripts/createComponent.js

```diff
@@ -34,6 +34,7 @@ const packageJson = () => {
   "name": "@govuk-react/${componentFolderName}",
   "version": "${version}",
   "dependencies": {
+    "@govuk-react/constants": "${version}",
```

Comment:
I think `^${version}` may be better, as it's ok to use an updated version of constants with non breaking changes

Reply:
but similarly, there may not be a version of constants matching the current lerna version, which I think is why I removed this, it's a bit tricky to know in advance what version would be needed

Reply:
should i take out this line?

Reply:
I reckon so